src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -224,9 +224,8 @@ void CommandRequestImpl::onTimeOut() {
                                             function_id(),
                                             correlation_id(),
                                             mobile_api::Result::GENERIC_ERROR);
-
+  (*response)[strings::msg_params][strings::info] = info;
   AddTimeOutComponentInfoToMessage(response);
```

Comment:
@pold500 Have you checked this method? Purpose of this method is exactly to create the "info" param for onTimeOut response, and currently if this method will run successfully, your received info parameter will be simply replaced inside of this method. So technically either you should update expected info inside of this method, or maybe there is something wrong with the script that it does not pass trough this method for some reason.